_posts/2020-11-19-Setting-Up-a-Pull-Request.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Setting Up a Pull Request
+description: 
+summary: 
+tags: 
+---
+Kajari instructed Celeste and I to start creating a new post using pull request. Pull requests are changes committed in the branch that are prresented to viewers of the users repository on Github. To initiate a new post/pull request, the following commands must be used to complete the task.
+* git checkout -b new post - the purpose of checkout -b is to create a new branch with the name new_post
+[This is the post of reference of other git commands after creating a new post](https://osvaldo178.github.io/2020/09/14/setting-up-git)
+
```

Comment:
Add in this command about getting github to recognize your new branch: https://github.com/celestem406/celestem406.Github.io/pull/2/files#r527143263